src/index.css

```diff
@@ -5,6 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #98969321;
 }
 
 code {
```

Comment:
Not sure what this line is doing here .... probably we should take a good look into that🤔